drivers/video/video_mcux_csi.c

```diff
@@ -293,7 +293,8 @@ static int video_mcux_csi_enqueue(const struct device *dev, enum video_endpoint_
 	}
 
 	to_read = data->csi_config.linePitch_Bytes * data->csi_config.height;
-	vbuf->bytesused = to_read;
+	vbuf->bytesframe = vbuf->bytesused = to_read;
+	vbuf->flags = VIDEO_BUF_EOF;
```

Comment:
Maybe other pull requests would be needed to propagate this change to the other drivers too?
So that it gets possible to rely on this from application/subsystem code.